modules/platforms/dotnet/Apache.Ignite.Core.Tests/ApiParity/TcpCommunicationSpiParityTest.cs

```diff
@@ -61,7 +61,8 @@ public class TcpCommunicationSpiParityTest
             "endTime",
             "id",
             "connectionIndex",
-            "NodeFilter"
+            "NodeFilter",
```

Comment:
Why this change?

Reply:
Because without such change some .NET [1] tests will be failed.
[1] https://ci.ignite.apache.org/viewLog.html?buildId=4387853&buildTypeId=IgniteTests24Java8_RunAll#testNameId5475589648862867137

Reply:
Seems, it should be reverted.